main.py

```diff
@@ -1,6 +1,6 @@
-from fastapi import FastAPI, status, HTTPException
-from fastapi.encoders import jsonable_encoder
+from fastapi import FastAPI, status, HTTPException, Query
 from pydantic import BaseModel, Field
+from sqlalchemy import asc, desc
```

Comment:
re-order the import as below ( in alphabetical order for each section)

1. Python standard libraries
2. installed libraries
3.  Methods from local files

![image](https://github.com/user-attachments/assets/4be04241-e882-4a97-86a7-efdc8eed0b40)